images/image.go

```diff
@@ -123,3 +105,28 @@ func (image *Image) Size(ctx context.Context, provider content.Provider) (int64,
 
 	}), image.Target)
 }
+
+// RootFS returns the unpacked diffids that make up and images rootfs.
+//
+// These are used to verify that a set of layers unpacked to the expected
+// values.
+func RootFS(ctx context.Context, provider content.Provider, desc ocispec.Descriptor) ([]digest.Digest, error) {
+	p, err := content.ReadBlob(ctx, provider, desc.Digest)
+	if err != nil {
+		log.G(ctx).Fatal(err)
```

Comment:
Looks like was like this before, but doubt it was intended to `Fatal` instead of return an error